src/main/java/tasks/Event.java

```diff
@@ -0,0 +1,64 @@
+package tasks;
+
+import java.util.Objects;
+
+public class Event extends Task {
+    private final String from;
+    private final String to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    public Event(String description, String from, String to, boolean completed) {
+        super(description, completed);
+        this.from = from;
+        this.to = to;
+    }
+
+    public String getFrom() {
+        return from;
+    }
+
+    public String getTo() {
+        return to;
+    }
+
+    @Override
+    public String getSaveFormat() {
+        return String.format("E | %d | %s | %s to %s", super.intComplete(), super.getDescription(), from, to);
+    }
+
+    @Override
+    public String toString() {
+        return String.format("[E][%s] %s (from: %s to: %s)", super.completedStringRepresentation(), super.getDescription(), from, to);
+    }
+
+    public static Event load(String input) {
+        String[] parameters = input.split("\\|");
+        String[] dateRange = parameters[3].split("to");
```

Comment:
Perhaps a better naming to include a plural noun instead so that it is clearly a collection of Strings?